components/derive/src/exonum_interface.rs

```diff
@@ -234,7 +286,7 @@ impl ExonumInterface {
                             let msg = format!("Method ID {} is already used", method_id);
```

Comment:
It could make sense to perform a similar check for removed method IDs. As far as I see, presently it's possible to define a method with the same ID as the removed one, and at best the compiler will complain about unreachable code in the method ID matcher (although I'm not sure such warnings are not suppressed for macro-generated code).

Reply:
My bad, I forgot to add it.